docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,424 @@ HTTP Response
       "status": 200
     }
 
+
+Creating a new User
+********************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
```

Comment:
```suggestion
This endpoint exposes the Ledger API's :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`.
```
`is a JSON API proxy for` -> `exposes` (simpler, the reader is already aware they are reading the JSON API documentation, no need to repeat it)
The documentation you are point to does not mention the user management service, so for the time being I'd rather not point to the wrong documentation if that has not been added to the Ledger API docs yet.

Reply:
Thanks for the input, I was already aware that this link isn't fully correct but I still had the urge to get some replacement (as described in my comment [here](https://github.com/digital-asset/daml/pull/12432#discussion_r786974026)). 

Besides I agree to your suggestion, however wanted to make you aware of that we've used the 
> is a JSON API proxy for

part in the documentation here already (which is why you find it here too because I based my draft here on the existing documentation).